Content.Client/Instruments/UI/InstrumentBoundUserInterface.cs

```diff
@@ -13,7 +12,7 @@ public sealed class InstrumentBoundUserInterface : BoundUserInterface
         public IEntityManager Entities => EntMan;
         [Dependency] public readonly IMidiManager MidiManager = default!;
         [Dependency] public readonly IFileDialogManager FileDialogManager = default!;
-        [Dependency] public readonly IPlayerManager PlayerManager = default!;
+        [Dependency] public new readonly IPlayerManager PlayerManager = default!;
```

Comment:
Can't you just remove this line entirely?

Reply:
On the presumption it was used, `IPlayerManager` is more specific than the parent `ISharedPlayerManager`, but looks like it's actually not used, so sure.